### PR TITLE
add markdown to resolution details in review step

### LINKED ISSUE #4203: have resolution details in review step support markdown

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/4203

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 👌

 This link will always point to this PR's latest changes: https://peek.run/vv2h2pk

File: packages/augur-ui/src/modules/common/form.styles.less [hunk 1]
```diff
@@ -1168,6 +1167,7 @@ div.FormDropdownActive {
 
     .SingleDatePicker_picker {
       margin-top: -1px;
+      z-index: @above-all-content;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.styles.less:1170]: Why mess with z-index?

File: packages/augur-ui/src/modules/common/form.styles.less [hunk 2]
```diff
@@ -1168,6 +1167,7 @@ div.FormDropdownActive {
 
     .SingleDatePicker_picker {
       margin-top: -1px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.styles.less:1169]: Also, recently John pointed out negative margin values are a code smell. Can you investigate if this negative is needed

File: packages/augur-ui/src/modules/common/form.styles.less (hunk 1 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/form.styles.less:1170]: The two date picker dropdowns were overlapping eachother

File: packages/augur-ui/src/modules/common/form.styles.less (hunk 2 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/form.styles.less:1169]: its needed so there isn't space below the arrow

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**